src/screens/Vault/SignerAdvanceSettings.tsx

```diff
@@ -552,6 +559,46 @@ function SignerAdvanceSettings({ route }: any) {
         return;
       }
 
+      if (SignersReqVault.includes(signer.type)) {
```

Comment:
Later we will want to do this also for other signers to show the change address in the transaction and the wallet it's from